backend/stack_overflow/src/main/java/com/gabrielcireap/stackOverflow/controller/AnswersController.java

```diff
@@ -0,0 +1,37 @@
+package com.gabrielcireap.stackOverflow.controller;
+
+import com.gabrielcireap.stackOverflow.dto.AnswerDTO;
+import com.gabrielcireap.stackOverflow.service.AnswerManagementService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class AnswersController {
+    private final AnswerManagementService answerManagementService;
+
+    @GetMapping("questions/{id}")
+    public List<AnswerDTO> findByQuestion(@PathVariable int id) {
+        return answerManagementService.findByQuestion(id);
+    }
+
+    @PostMapping("/answers")
+    public AnswerDTO addAnswer(@RequestBody AnswerDTO answerDTO){
+        return answerManagementService.save(answerDTO.getQuestion().getId(), answerDTO.getText());
+    }
+
+    @PutMapping("/answers")
```

Comment:
Usually for update operations you also have the ID in the URL (even though it is a little redundant as you probably have it in the body).